latex/corpusmakros.sh

```diff
@@ -0,0 +1,43 @@
+
+# php files + loc
+PHPFILES=$(cd ../phpfiles; ls -1 | wc -l | xargs printf "%'d")
+PHPLOC=$(find ../phpfiles/ -name '*.php' | xargs wc -l | grep total | tr -d \[:alpha:\] | tr -d \[:space:\] | xargs printf "%'d")
+# java files + loc
+JAVAFILES=$(cd ../javastdlib5; find . -name "*.java" | wc -l | xargs printf "%'d")
+JAVALOC=$(find ../javastdlib5/ -name '*.java' | xargs wc -l | grep "total" | tr -d \[:alpha:\] | paste -sd+ | bc | xargs printf "%'d")
+# lua files + loc
+LUAFILES=$(cd ../lua/testes/; find . -name "*.lua" | wc -l | xargs printf "%'d")
+LUALOC=$(find ../lua/testes/ -name '*.lua' | xargs wc -l | grep "total" | tr -d \[:alpha:\] | paste -sd+ | bc | xargs printf "%'d")
+# sqlfiles + test + loc
+SQLFILES=$(cd ../sqlite/test/; find . -name "*.test" | wc -l | xargs printf "%'d")
+SQLTESTS=$(cd ../sqlfiles/; find . -name "*.sql" | wc -l | xargs printf "%'d")
+SQLLOC=$(find ../sqlfiles/ -name '*.sql' | xargs sed s/
/\\n/g | wc -l | xargs printf "%'d")
```

Comment:
Is the `^M` character intentional? Is it equivalent to `\\r`?

Reply:
As far I as I can tell no. The sqlitest seem to use dos endings and this gets rid of them.

Reply:
`^M` is `13` in disguise and `\r` is normally the nice way of representing that character. Basically I'm nervous about control codes like `^M` in files, so if `\\r` does the same thing here (which I think it might) it would be preferable.